src/scripts/gym/Gym.ts

```diff
@@ -61,12 +61,12 @@ gymList['Pewter City'] = new Gym(
     'Brock',
     'Pewter City',
     [
-        new GymPokemon('Geodude', 770, 12),
-        new GymPokemon('Onix', 1554, 14),
+        new GymPokemon('Geodude', 770, 10),
+        new GymPokemon('Onix', 1554, 12),
     ],
     BadgeEnums.Boulder,
     250,
-    'I took you for granted, and so I lost. As proof of your victory, I confer on you this...the official Pokémon League Boulder Badge.',
+    'I took you for granted. As proof of your victory, here\'s the BoulderBadge!',
```

Comment:
```suggestion
    'I took you for granted. As proof of your victory, here\'s the Boulder Badge!',
```

Since the badge case displays them with a space, I think it makes more sense to have the gym leaders use a space in the badge names too.